packages/react-components/react-table/src/components/Table/useTableStyles.ts

```diff
@@ -16,7 +17,7 @@ const useStyles = makeStyles({
     borderCollapse: 'collapse',
     width: '100%',
     display: 'table',
-    tableLayout: 'fixed',
```

Comment:
Lets the browser layout the table by default, this can be added quite easily by users for custom layout through stlye override